### DISCUSSION TITLE
Profile Review Pending

### DISCUSSION THREAD

**Discussion by rungta-ayush** [github/india#693] [Students]
Hello Team,
I finished my profile today and submitted it for review today. Until now, it is still under review and I haven't received any update on the same.
With the deadline being today (midnight), I request you to please fast-track the process so that I could submit my proposal.
Any kind of assistance/help would be really appreciated.

**UPDATE : [SOLVED]**

**Comment by rungta-ayush** [comment]
Only 1.5 hours left.
Awaiting for a response!

**Reply by raaghavrm** [reply]
did you got a solution bro ?

**Reply by rungta-ayush** [reply]
Yes!

**Comment by d-riddle** [comment]
Same is the case with me. I also finished my profile today and submitted it for review today. It's still under review.
Any help would be appreciated.

**Reply by raaghavrm** [reply]
did you got a solution bro ?
